internal/crypto/tokens.go

```diff
@@ -1,35 +1,29 @@
 package crypto
 
 import (
-	"crypto/ecdsa"
 	"errors"
-	"fmt"
 
 	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
 	"github.com/nspcc-dev/neofs-sdk-go/user"
 )
 
 // TODO: https://github.com/nspcc-dev/neofs-node/issues/2795 after API stabilization, move some components to SDK
 
-// SignedToken is a common interface of signed NeoFS tokens.
-type SignedToken interface {
```

Comment:
`SignedEntity` then? Not sure why it's removed completely.

Reply:
used once now